app/views/map/index.html.erb

```diff
@@ -1 +1,243 @@
-<div class="map"></div>
+<!DOCTYPE html>
+<html>
+  <head>
+  	<title>HPI Map</title>
+	<meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
+  	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.3.1/leaflet.css" />
+    <script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js" integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA==" crossorigin=""></script>
```

Comment:
If we use leaflet purely in JS as we currently do, I'd be in favor of using npm for package management instead of importing scripts directly. To do this, execute `yarn add leaflet leaflet.locatecontrol` , and add the following lines to app/javascript/packs/application.js:
```javascript
import L from 'leaflet';
import 'leaflet.locatecontrol'
```
and remove the script src tags from map/index.html.erb. The CSS imports need to be kept

Reply:
done